src/foam/nanos/ruler/test/DummyErroneousPredicate.java

```diff
@@ -0,0 +1,14 @@
+package foam.nanos.ruler.test;
+
+import foam.mlang.predicate.AbstractPredicate;
+
+/**
+ * DummyErroneousPredicate - always throws exception when evaluated.
```

Comment:
What is this for?

Reply:
It's for simulating a predicate that throws exception and being used for testing RulerDAO - https://github.com/lchanmann/foam2/blob/SmallFixRuleEngine/src/foam/nanos/ruler/test/RulerDAOTest.java#L211.